include/stronk/stronk.h

```diff
@@ -6,62 +6,78 @@
 #include <utility>
 
 #include <stronk/utilities/equality.h>
+#include <stronk/utilities/macros.h>
 #include <stronk/utilities/strings.h>
 
 namespace twig
 {
+
+template<typename T>
+concept should_be_copy_constructed = std::is_trivially_copyable_v<T> && sizeof(T) <= sizeof(T*);
```

Comment:
Would `sizeof(void*)` make it clearer that this is just the pointer size on the system? If I have understood the code correctly